_layouts/base.html

```diff
@@ -12,11 +12,15 @@
   <link href="https://fonts.googleapis.com/css?family=Bitter:400i" rel="stylesheet">
   <!--load asap-->
   <link href="https://fonts.googleapis.com/css?family=Asap:400,600" rel="stylesheet">
+  <!-- this is typer -->
+  <link rel="stylesheet" href="https://unpkg.com/typer-js/typer.css">
 </head>
 
 <body>
   {{ content }}
 
+  
+  <script src="https://unpkg.com/typer-js"></script>
```

Comment:
you don't need this script if you are importing Typer from npm

Reply:
leftover from typer.js before switch to typed.js